torchdynamo/guards.py

```diff
@@ -91,7 +100,19 @@ def __lt__(self, other):
         return self.sort_key() < other.sort_key()
 
     def __str__(self):
-        return f"{self.source.name.lower()} {repr(self.name)} {self.create_fn.__name__}"
+        s = f"{self.source.name.lower()} {repr(self.name)} {self.create_fn.__name__}"
+        if config.export_guards is True:
```

Comment:
```suggestion
        if config.export_guards:
```

This in this case we could just check if the actual values are `None` and may not need a config.